PR-TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# PR name
+
+* Related Issues : _#IssueNumber if necessary_
+* Previous PRs: _#PRNumber if this PR is a follow-up_
+
+## Introduction
+
+A short description of what the feature is. Try to keep it to a
+single-paragraph "elevator pitch" so the reader understands what
+problem this proposal is addressing.
+
+## Motivation
+
+Describe the problems that this proposal seeks to address. If the
+problem is that some common pattern is currently hard to express, show
+how one can currently get a similar effect and describe its
+drawbacks. If it's completely new functionality that cannot be
+emulated, motivate why this new functionality is necessary.
+
+## Proposed solution
+
+Describe your solution to the problem. Provide examples and describe
+how they work. Show why your solution is better than current
+workarounds: is it cleaner, safer, or more efficient?
+
+## Current PR Issues
+
+If present describe some issues in the Pull Request, in order to help others in the full completition of the PR
```

Comment:
I'm pretty sure you don't mean "completition" here, but rather "resolution" or some other synonym.

Also missing a period at the end of this sentence.